test.md

```diff
@@ -0,0 +1 @@
+this file should be ignored.
```

Comment:
- [ ] Kindly make sure the `test.md` file is ignored by git so it means it must be added into the `gitignore` file and should not be present in the repo in order to match the project requirement. Kindly fix it.